king_phisher/client/windows/campaign_import.py

```diff
@@ -238,17 +238,41 @@ def select_xml_campaign(self):
 			campaign_xml = ET.parse(target_file)
 		except ET.ParseError as error:
 			self.logger.error("cannot import campaign: {0} is not a valid XML file".format(target_file), error)
-			raise KingPhisherInputValidationError("{0} is not a valid xml file".format(target_file))
+			gui_utilities.show_dialog_error(
+				'Improper Format',
+				self.window,
+				'{} is not a valid XML file'.format(target_file)
+			)
+			return
 
 		root = campaign_xml.getroot()
 		if root.tag != 'king_phisher':
-			raise KingPhisherInputValidationError('File not a King Phisher Campaign XML Export')
+			self.logger.error("not a King Phisher XML campaign file: {}".format(target_file))
```

Comment:
This should probably be the same message as `L240` and the reasoning would look better in quotes like: `cannot import campaign file: {0} (not a valid xml file)`. there and `... (invalid root node)` here. This makes it obvious that the errors are related but the specific reasons are different.